src/variant/transform2d.cpp

```diff
@@ -237,7 +239,7 @@ real_t Transform2D::basis_determinant() const {
 }
 
 Transform2D Transform2D::interpolate_with(const Transform2D &p_transform, real_t p_c) const {
-	//extract parameters
+	// extract parameters
```

Comment:
While at it, it should use our comment style: https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#comment-style-guide
```suggestion
	// Extract parameters.
```
Same for others.